lib/dor/text_extraction/abbyy/results.rb

```diff
@@ -42,11 +52,43 @@ def alto_doc
           output_docs['alto']
         end
 
```

Comment:
This does the work of *moving* the Abbyy results files, including split out OCR files, to another location (which will be the workspace directory).